config.go

```diff
@@ -12,23 +14,54 @@ type Config struct {
 	Packages map[string]Package `yaml:"packages"`
 }
 
+// orderedConfig is used internally to check for alphabetical ordering in the YAML
+// configuration. A yaml.MapSlice perservers ordering of keys: https://godoc.org/gopkg.in/yaml.v2#MapSlice
+type orderedConfig struct {
+	URL      string        `yaml:"url"`
+	Packages yaml.MapSlice `yaml:"packages"`
+}
+
 // Package details the options available for each repo
 type Package struct {
 	Repo string `yaml:"repo"`
 }
 
+// ensureAlphabetical checks that the packages are listed alphabetically in the configuration.
+func ensureAlphabetical(data []byte) bool {
+	var c orderedConfig
```

Comment:
Also, since this is used only here, I would inline the struct. Plus since the
URL is not actually used, you can just do this instead:

```go
var c struct{ Packages yaml.MapSlice `yaml:"packages"` }
if err := yaml.Unmarshal(data, &c); err != nil {
    return false
}
```